docs/code-examples/depth_image_3d.py

```diff
@@ -20,4 +20,4 @@
 )
 
 # Log the tensor, assigning names to each dimension
-rr.log_depth_image("world/camera/depth", image, meter=10000.0)
+rr.log_depth_image("world/camera/depth", image, tensor_id=np.repeat(0, 16).astype(np.uint8), meter=10000.0)
```

Comment:
```suggestion
rr.log_depth_image("world/camera/depth", image, meter=10000.0)
```

I ran into this on the other image/tensor round-trip-tests as well.

This one is particularly bad. If a user copies this blindly it it's actually going to cause problems for them if they end up logging multiple tensors with the same id. 

I don't think we should leak this into our code-examples. We should either opt this one out of round-tripping or, better yet, we should implement a an exception for TensorId in the round-trip test framework.